docs/strategies/elixir.md

```diff
@@ -0,0 +1,136 @@
+# Elixir Analysis
+
+When developing in Elixir, [Mix](https://hexdocs.pm/mix/Mix.html) and [Hex](https://hex.pm/) are most commonly used to manage dependencies. 
+
+| Strategy | Direct Deps        | Deep Deps          | Edges              |
+| -------- | ------------------ | ------------------ | ------------------ |
+| mix deps | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+
+## Project Discovery
+
+In order to find elixir projects, we look for `mix.exs` file which specify the root of a elixir project. We skip `deps` and `_build` directories. 
+
+## Analysis
+
+1. Run `mix deps.tree --format plain` and generate output similar to:
```

Comment:
```suggestion
1. Run `mix deps.tree --format plain --only prod` and generate output similar to:
```

Reply:
nvm, this is already done, we were discussing an old commit after you had fixed it, my bad.